WebRoot/public/extuserprov.jsp

```diff
@@ -77,37 +76,44 @@
 				</table>
 			</div>
 
-			<table class="form">
-				<c:choose>
-				<c:when test="${not empty domainLoginRedirectUrl && param.sso eq 1 && empty param.ignoreLoginURL && (isAllowedUA eq true)}">
-					<tr>
-						<td colspan="2">
-							<div class="LaunchButton">
-								<input type="submit" value="<fmt:message key="launch"/>" >
-							</div>
-						</td>
-					</tr>
-				</c:when>
-				<c:otherwise>
-					<tr>
-						<td><label for="displayname"><fmt:message key="displayName"/>:</label></td>
-						<td><input id="displayname" class="zLoginField" name="displayname" type="text" value="${fn:escapeXml(param.displayname)}" size="40" maxlength="${domainInfo.webClientMaxInputBufferLength}"/></td>
-					</tr>
-					<tr>
-						<td><label for="password"><fmt:message key="password"/>:</label></td>
-						<td><input id="password" class="zLoginField" name="password" type="password" value="${fn:escapeXml(param.password)}" size="40" maxlength="${domainInfo.webClientMaxInputBufferLength}"/></td>
-					</tr>
-					<tr id="confirmPassword" style="display:none">
-						<td><label for="password2"><fmt:message key="confirm"/>:</label></td>
-						<td><input id="password2" class="zLoginField" name="password2" type="password" size="40" maxlength="${domainInfo.webClientMaxInputBufferLength}"/></td>
-					</tr>
-					<tr>
-						<td>&nbsp;</td>
-						<td style="text-align:right">
-							<input type="submit" class="zLoginButton" value="<fmt:message key="register"/>" style="float:left;"/>
-					</tr>
-				</c:otherwise>
-				</c:choose>
+			<table class="modernForm">
+				<div class="loginSection">
+					<c:choose>
+					<c:when test="${not empty domainLoginRedirectUrl && param.sso eq 1 && empty param.ignoreLoginURL && (isAllowedUA eq true)}">
+						<tr>
+							<td colspan="2">
+								<div class="LaunchButton">
+									<input type="submit" value="<fmt:message key="launch"/>" >
+								</div>
+							</td>
+						</tr>
+					</c:when>
+					<c:otherwise>
+						<tr>
+							<td><label class="zLoginFieldLabel" for="displayname"><fmt:message key="displayName"/></label></td>
+						</tr>
+						<tr>
+							<td><input id="displayname" class="zLoginField" name="displayname" type="text" value="${fn:escapeXml(param.displayname)}" size="40" maxlength="${domainInfo.webClientMaxInputBufferLength}"/></td>
+						</tr>
+						<tr>
+							<td><label class="zLoginFieldLabel" for="password"><fmt:message key="password"/></label></td>
+						</tr>
+						<tr>
+							<td><input id="password" class="zLoginField" name="password" type="password" value="${fn:escapeXml(param.password)}" size="40" maxlength="${domainInfo.webClientMaxInputBufferLength}"/></td>
+						</tr>
```

Comment:
for type="password" use ` class=zLoginFieldInput ` have a look at the login page , try to make it consistent

Reply:
Done